.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
+sudo: required
+services:
+  - docker
+
+install:
+  - docker build -rm -t buildenv docker/
+
+script:
+  - docker run -t --rm -v `pwd`:/io buildenv /opt/python/3.5.1/bin/pip3 install numpy -vv
```

Comment:
For optimistic future-proofing, maybe add `--no-binary :all:` to the pip command here :-)